HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -1,19 +1,32 @@
 import FWCore.ParameterSet.Config as cms
 
-# CMSSW version specific customizations
-import os
-cmsswVersion = os.environ['CMSSW_VERSION']
-
-if cmsswVersion >= "CMSSW_7_5":
 
 # Simplified TrackerTopologyEP config (PR #7966)
-    if 'trackerTopology' in locals():
-        paras = tuple(vars(trackerTopology)['_Parameterizable__parameterNames'])
-        for para in paras:
-          delattr(trackerTopology,para)
-        setattr(trackerTopology,'appendToDataLabel',cms.string(""))
+def customiseFor7966(process):
+    if hasattr(process, 'trackerTopology'):
+        params = process.trackerTopology.parameterNames_()
+        for param in params:
+            delattr(process.trackerTopology, param)
+        setattr(process.trackerTopology, 'appendToDataLabel', cms.string(""))
+    return process
+
 
 # Removal of 'upgradeGeometry' from TrackerDigiGeometryESModule (PR #7794)
-    if 'TrackerDigiGeometryESModule' in locals():
-        if  hasattr(TrackerDigiGeometryESModule.trackerGeometryConstants,'upgradeGeometry'):
-            delattr(TrackerDigiGeometryESModule.trackerGeometryConstants,'upgradeGeometry')
+def customiseFor7794(process):
+    if hasattr(process, 'TrackerDigiGeometryESModule'):
+        if hasattr(process.TrackerDigiGeometryESModule, 'trackerGeometryConstants'):
+            if hasattr(process.TrackerDigiGeometryESModule.trackerGeometryConstants, 'upgradeGeometry'):
+                delattr(process.TrackerDigiGeometryESModule.trackerGeometryConstants, 'upgradeGeometry')
+    return process
+
+
+# CMSSW version specific customizations
+def customise(process):
+    import os
+    cmsswVersion = os.environ['CMSSW_VERSION']
```

Comment:
In principle I agree with David: this file can easily be release-dependent, so there would be no need to have explicit checks on `CMSSW_VERSION` .
However, I leave it up to @Martin-Grunewald and @perrotta to decide how they prefer to work.